src/libraries/Directory.Build.props

```diff
@@ -245,6 +244,9 @@
     <!-- Always pass portable to override arcade sdk which uses embedded for local builds -->
     <DebugType>portable</DebugType>
 
+    <!-- Microsoft.Extensions projects have a separate StrongNameKeyId -->
+    <StrongNameKeyId Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.'))">MicrosoftAspNetCore</StrongNameKeyId>
```

Comment:
Why here and not at line 154?

Reply:
It has to go somewhere after line https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.props#L238 otherwise the property would get reassigned with MicrosoftShared by:
https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L6

Reply:
I see, so line 154 does nothing.  🤦‍♂   We should file a bug on that.  I wonder if that caused other projects to get a different public key.

Reply:
I don't think so, because other projects have each their own Directory.Build.props next to their solution files which specifies the StrongNameKeyId. e.g.: 
- https://github.com/dotnet/runtime/blob/master/src/libraries/System.Globalization/Directory.Build.props#L4
- https://github.com/dotnet/runtime/blob/master/src/libraries/System.Data.Odbc/Directory.Build.props#L4

Reply:
But the intent of that line was to make the default for corefx (at the time) to be the Open key. If it’s not having any effect that means we may have some projects which relied on it but no longer get it.  IOW the didn’t set Open key in their d.b.p. As it stands it does nothing and should be deleted.